CefSharp/Internals/JavascriptObjectRepository.cs

```diff
@@ -314,6 +317,14 @@ internal bool TryCallMethod(long objectId, string name, object[] parameters, out
                     }
                     else
                     {
+                        // anyone extending the Method Interceptor should be able to use Task 
+                        if (obj.MethodInterceptor.IsAsynchronous)
```

Comment:
> @Codeusa I'll make my proposed code changes and push them to a branch for further discussion. Hopefully in the next day or so I'll have something to discuss 👍

Thanks for the review! I will look over your comments in the next few hours and reply. I also apologize if any of my comments inline came off as opinionated (bumping version, etc.)

Reply:
> . I also apologize if any of my comments inline came off as opinionated (bumping version, etc.)

Totally fine 😄 Hopefully my response came across as diplomatic, personally I wish bumping version was a simple process that everyone would be onboard with. When I upgraded from `.Net 4.0` to `.Net 4.5.2` i copped quite a bit of flack from people. I'm hoping to avoid comments like https://github.com/cefsharp/CefSharp/issues/1785#issuecomment-299615017 if can be helped 😦 

When I can find a few minutes I'll try install the `Microsoft.Net.Compilers` packages again, at least we'll be able to use the newer syntax if I can get it building.